internal/updater/utils/html/parsing.go

```diff
@@ -0,0 +1,70 @@
+package utils
+
+import (
+	"errors"
+
+	"github.com/andybalholm/cascadia"
+	"golang.org/x/net/html"
+)
+
+var (
+	ErrAttrNotFound = errors.New("matching attribute not found")
+)
+
+func GetText(n *html.Node) string {
+	return n.FirstChild.Data
+}
+
+func GetAttr(n *html.Node, key string) (string, error) {
```

Comment:
nit 

That should be explicit enough to not require a comment :thinking: 

```suggestion
func GetAttributeValue(node *html.Node, attributeKey string) (attributeValue string, err error) {
```

Reply:
Fixed